helper/include/odp/helper/cli.h

```diff
@@ -0,0 +1,84 @@
+/* Copyright (c) 2021, Nokia
+ * All rights reserved.
+ *
+ * SPDX-License-Identifier:     BSD-3-Clause
+ */
+
+/**
+ * @file
+ *
+ * ODP CLI helper API
+ *
+ * This API allows control of ODP CLI server, which may be connected to
+ * using a telnet client. CLI commands may be used to get information
+ * from an ODP instance, for debugging purposes.
+ *
+ * Many CLI commands output the information to the console, or wherever
+ * ODP logs have been directed to in global init.
+ */
+
+#ifndef ODPH_CLI_H_
+#define ODPH_CLI_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <odp_api.h>
+#include <stdint.h>
+#include <netinet/in.h>
+
+/**
+ * @addtogroup odph_cli ODPH CLI
+ * @{
+ */
+
+/** ODP CLI server parameters. */
+typedef struct {
+	/** A character string containing an IP address. */
+	char address[INET6_ADDRSTRLEN];
```

Comment:
I'd like to avoid having to include the `netinet/in.h` header. Instead, this be `const char *` or `ODPH_IPV6ADDRSTR_LEN` could be defined in `helper/include/odp/helper/ip.h.

Reply:
I'm putting the defines (for both v4 and v6) in ip.h. Trying to avoid passing pointers, although it probably doesn't matter much.